Resources/config/auto_route.xml

```diff
@@ -86,6 +86,14 @@
             class="%cmf_routing_auto.metadata.factory.class%"
             factory-service="cmf_routing_auto.metadata.factory.builder"
             factory-method="getMetadataFactory"
-        />
+            />
+
+        <!-- Controller -->
+        <service 
+            id="cmf_routing_auto.redirect_controller"
+            class="Symfony\Cmf\Bundle\RoutingAutoBundle\Controller\RedirectController"
```

Comment:
It's here: http://symfony.com/doc/current/best_practices/business-logic.html#service-no-class-parameter

Anyway, that's going to be rewritten as the class parameters are going to be removed as there are better ways. See the issue @dbu referenced